tasks/65_smaller-common-multiple/smaller-common-multiple.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * smallestCommons - finds the smallest common multiple of the
+ * provided parameters that can be evenly divided by both, as
+ * well as by all sequential numbers in the range between these
+ * parameters
+ * @example
+ * // return 60
+ * smallestCommons([1, 5]);
+ * @param {Array<Number>} arr
+ * @return {Number}
+ */
+export function smallestCommons(arr) {
+  const max = Math.max(...arr);
+  const min = Math.min(...arr);
+  const range = getRange(min, max);
+  let currentLCM = lcm(range[0], range[1]);
+  for (let i = 1; i < range.length; i++) {
+    currentLCM = lcm(currentLCM, range[i]);
+  }
+  return currentLCM;
+}
+
+/**
+ * gcd - finds greatest common division of two numbers
+ * @example
+ * // return 3
+ * gcd(15, 12);
+ * @param {Number} a
+ * @param {Number} b
+ * @return {Number}
+ */
+function gcd(a, b) {
```

Comment:
Try to avoid abbreviation if it's not common/conventional abbreviation. Don't try to save letters, code readability is more important.